packages/design-system/.storybook/theme.js

```diff
@@ -1,38 +1,55 @@
 import { create } from '@storybook/theming/create';
-import logo from './logo.svg';
-
-export default create({
-	base: 'light',
-
-	colorPrimary: '#0675C1',
-	colorSecondary: '#19426c',
-
-	// UI
-	appBg: '#f3f3f3',
-	appContentBg: 'white',
-	appBorderColor: '#C6C6C6',
-	appBorderRadius: 4,
-
-	// Typography
-	fontBase: '"Source Sans Pro", sans-serif',
-	fontCode: '"Inconsolata", monospace',
-
-	// Text colors
-	textColor: '#202020',
-	textInverseColor: 'rgba(255,255,255,0.9)',
-
-	// Toolbar default and active colors
-	barTextColor: '#555555',
-	barSelectedColor: '#202020',
-	barBg: 'white',
-
-	// Form colors
-	inputBg: 'white',
-	inputBorder: 'silver',
-	inputTextColor: 'black',
-	inputBorderRadius: 4,
-
-	brandTitle: 'Coral',
-	// brandUrl: '/',
-	brandImage: logo,
-});
+
+import dictionaryLight from '@talend/design-tokens/lib/light/dictionary';
+import dictionaryDark from '@talend/design-tokens/lib/dark/dictionary';
+
+import logoLight from './logo-light.svg';
+import logoDark from './logo-dark.svg';
+
+export default mode => {
+	const isDark = mode === 'dark';
+	const dictionary = isDark ? dictionaryDark : dictionaryLight;
+	const logo = isDark ? logoDark : logoLight;
+	const t = tokenName => {
+		const token = dictionary.find(t => t.name === tokenName);
+		if (!token) {
+			throw Error('Cannot find token ' + tokenName);
+		}
+		return token.value;
+	};
+	return create({
```

Comment:
Storybook themes can't rely on CSS custom properties. 
That's why we build a theme with hard-coded colors and co.